src/x11api.h

```diff
@@ -7,17 +7,19 @@
 #define MASK_CONFIG_MOUSE (0)
 #define MASK_CONFIG_KEYBOARD (1)
 
-#define CLICK_MODE_XTEST (0) // Default
-#define CLICK_MODE_XEVENT (1)
+enum ClickModes {
+    CLICK_MODE_XTEST,
+    CLICK_MODE_XEVENT,
+};
 
 /**
  * @brief Enumeration of the supported mouse events.
  * @see mouse_event
  */
 enum MouseEvents
 {
-    MOUSE_PRESSED,
-    MOUSE_RELEASED
+    MOUSE_EVENT_PRESS,
+    MOUSE_RELEASE,
```

Comment:
Feels a bit weird to me. Why there is a `_EVENT_` part for press but not for release?

Reply:
Thank you for pointing that out. Didn't notice it.